app/src/main/java/org/kiwix/kiwixmobile/utils/ServerUtils.java

```diff
@@ -0,0 +1,60 @@
+package org.kiwix.kiwixmobile.utils;
+
+import androidx.annotation.NonNull;
+import java.net.InetAddress;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.util.Enumeration;
+
+public class ServerUtils {
+  public static int port;
+  public static boolean isServerStarted;
+
+  // get Ip address of the device's wireless access point i.e. wifi hotspot OR wifi network
+  public static String getIpAddress() {
+    String ip = "";
+    try {
+      Enumeration<NetworkInterface> enumNetworkInterfaces = NetworkInterface
+          .getNetworkInterfaces();
+      while (enumNetworkInterfaces.hasMoreElements()) {
+        NetworkInterface networkInterface = enumNetworkInterfaces
+            .nextElement();
+        Enumeration<InetAddress> enumInetAddress = networkInterface
+            .getInetAddresses();
+        while (enumInetAddress.hasMoreElements()) {
+          InetAddress inetAddress = enumInetAddress.nextElement();
+
+          if (inetAddress.isSiteLocalAddress()) {
+            ip += inetAddress.getHostAddress() + "\n";
```

Comment:
why do you add `\n` when you always replace it?

Reply:
Fixed e30aa7b78c98ca53338fa1392808bc770cd7255f